common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js

```diff
@@ -28,12 +28,7 @@ define('video/09_play_pause_control.js', [], function() {
             '<button class="control video_control play" aria-disabled="false" title="',
                 gettext('Play'),
             '">',
-                '<span class="icon-fallback-img">',
-                    '<span class="icon fa fa-play" aria-hidden="true"></span>',
-                    '<span class="sr control-text">',
```

Comment:
Since you are removing these control-text spans, you also need to remove their associated JavaScript.

````
pause: function () {
            this.el
                .removeClass('pause')
                .addClass('play')
                .attr('title', gettext('Play'))
                .find('.icon')
                    .removeClass('fa-pause')
                    .addClass('fa-play');

            this.el
                .find('.control-text')
                    .text(gettext('Play'));
        }
```
Search through all the video code to make sure the spans you are removing don't have JS that also needs to be cleaned up.